index.js

```diff
@@ -36,12 +40,20 @@ const wrapWord = (rows, word, columns) => {
 
 		if (ESCAPES.has(character)) {
 			isInsideEscape = true;
-		} else if (isInsideEscape && character === 'm') {
-			isInsideEscape = false;
-			continue;
+			isInsideLinkEscape = word.slice(index).startsWith(`${character}${ANSI_ESCAPE_LINK}`);
 		}
 
 		if (isInsideEscape) {
+			/* istanbul ignore if: cannot enter on terminals not supporting hyperlinks */
```

Comment:
The code doesn't take into account terminal capabilities, so why is this ignored for coverage?